ethstore/src/dir/disk.rs

```diff
@@ -226,7 +233,7 @@ impl<T> KeyDirectory for DiskDirectory<T> where T: KeyFileManager {
 	}
 
 	fn unique_repr(&self) -> Result<u64, Error> {
-		self.files_hash()
+		self.last_modification_date()
```

Comment:
seems like it's only a unique representation if
  - the filesystem stores modification times with very high precision
  - no two files have the same modification time